onnxruntime/core/optimizer/transpose_optimizer/optimizer_api_impl.cc

```diff
@@ -828,14 +828,14 @@ onnxruntime::Node& NodeFromApiNode(onnx_layout_transformation::api::NodeRef& nod
 namespace layout_transformer {
 
 const std::unordered_set<std::string_view>& GetORTLayoutSensitiveOps() {
-  static std::unordered_set<std::string_view> ort_layout_senstive_ops = []() {
+  static std::unordered_set<std::string_view> ort_layout_sensitive_ops = []() {
     const auto& layout_sensitive_ops = onnx_layout_transformation::GetLayoutSensitiveOps();
-    std::unordered_set<std::string_view> ort_specific_ops = {"Resize", "FusedConv", "QLinearAveragePool", "QLinearGlobalAveragePool"};
+    std::unordered_set<std::string_view> ort_specific_ops = {"FusedConv", "QLinearAveragePool", "QLinearGlobalAveragePool"};
     ort_specific_ops.insert(layout_sensitive_ops.cbegin(), layout_sensitive_ops.cend());
```

Comment:
This needs to match transpose_optimizer.cc and only include Resize if we do not add the resize_handler. In that scenario Resize is assumed to be NCHW specific.

`#if !defined(USE_CUDA) && !defined(USE_ROCM)`

Reply:
Thanks for pointing out.